### PR TITLE
[Merge Script Test] Except not merged: conflict (Tested Review Comments)

### PR DESCRIPTION
**Pull request by yifanr**

### CODE REVIEW

**Comment by YifanRen** [comment]
comment from a second person

**Comment by yifanr** [comment]
Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
ops, one more comment!

**Comment by YifanRen** [comment]
update

**Comment by yifanr** [comment]
This pull request is not merged: Merge attampt failed: Pull Request is not mergeable

File: file7 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+8
+88
+888
```
Comment (YifanRen) [inline file7]: review comments

**Comment by yifanr** [comment]
This pull request is not merged: CircleCI Test failed. Please fix the test failure.

**Comment by yifanr** [comment]
This pull request is not merged: Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
This pull request is not merged: Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
This pull request is not merged: Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
This pull request is not merged: Merge attampt failed: Pull Request is not mergeable

**Comment by yifanr** [comment]
This pull request is not merged: CircleCI Test failed. Please fix the test failure.

**Comment by yifanr** [comment]
This pull request is not merged: Feature branch is stale. Please rebase.

**Comment by YifanRen** [comment]
update

**Comment by yifanr** [comment]
This pull request is not merged: CircleCI Test failed. Please fix the test failure.

**Comment by YifanRen** [comment]
update

**Comment by yifanr** [comment]
This pull request is not merged: Merge attampt failed: Pull Request is not mergeable

**Comment by YifanRen** [comment]
update

**Pull request merged**